andela/titan-dev-ah-frontend
# #163518661 create bookmark articles component

#### What does this PR do?
Implement the functionality which allows users to bookmark an article.
#### Description of Task to be completed?
- Convert bookmark icon o react component
- Create bookmark actions creators
- Create bookmark reducers
- Bind onClick event to bookmark icons 
- Test  actions creators, reducers, components and events
- Add bookmark component to read article page and all components

#### How should this be manually tested?
- Clone the repo and check out the following branch `ft-bookmark-articles-163518661`
- Run `yarn start` and  open an article
- Click to bookmark icon and wait for the action to be executed. The icon should change the color.
- Run `yarn test` to run the unit tests

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163518661
#### Screenshots (if appropriate)
#### Remark:
-  Should wait for  `#165352853` PR to be merged to be marked as completed.
The bookmark component should be integrated on the read article page.
